pandas/core/arrays/sparse/scipy_sparse.py

```diff
@@ -19,76 +35,78 @@ def _check_is_partition(parts, whole):
         raise ValueError("Is not a partition because union is not the whole.")
 
 
-def _to_ijv(ss, row_levels=(0,), column_levels=(1,), sort_labels=False):
+def _levels_to_axis(
+    ss,
+    levels: Tuple[int] | List[int],
+    valid_ilocs: np.ndarray,
+    sort_labels: bool = False,
+) -> Tuple[npt.NDArray[np.intp], List[IndexLabel]]:
+    """
+    For a MultiIndexed sparse Series `ss`, return `ax_coords` and `ax_labels`,
+    where `ax_coords` are the coordinates along one of the two axes of the
+    destination sparse matrix, and `ax_labels` are the labels from `ss`' Index
```

Comment:
can you add Parmeters / Returns sections

Reply:
done